{"uuid": "7f98328b-15de-41e0-89e6-8e5cbc950d4d", "children": ["c07e79a7-0100-4ed6-b61c-0db98e0bebfc"], "befores": [{"name": "get_driver", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectionError: Could not reach host. Are you offline?\n", "trace": "  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\PycharmProjects\\pythonProject\\conftest.py\", line 28, in get_driver\n    service = Service(executable_path=ChromeDriverManager().install())\n                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\webdriver_manager\\chrome.py\", line 40, in install\n    driver_path = self._get_driver_binary_path(self.driver)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\webdriver_manager\\core\\manager.py\", line 40, in _get_driver_binary_path\n    file = self._download_manager.download_file(driver.get_driver_download_url(os_type))\n                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\webdriver_manager\\drivers\\chrome.py\", line 45, in get_driver_download_url\n    modern_version_url = self.get_url_for_version_and_platform(driver_version_to_download, os_type)\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\webdriver_manager\\drivers\\chrome.py\", line 75, in get_url_for_version_and_platform\n    response = self._http_client.get(url)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Serjan777\\miniconda3\\Lib\\site-packages\\webdriver_manager\\core\\http.py\", line 35, in get\n    raise exceptions.ConnectionError(f\"Could not reach host. Are you offline?\")\n"}, "start": 1732721128322, "stop": 1732721130790}], "start": 1732721128322, "stop": 1732721131925}